### ISSUE TITLE
Packages: Resolve aliased packages to their upstream versions

### ISSUE THREAD

**Issue by saramarcondes** [ItsJonQ/g2#124]
`a11y`, `gestures` and `animations` are currently aliased packages that simply re-export upstream dependencies. Before integrating G2 these packages should be re-resolved to their upstream versions rather than the aliased versions.

- [x] `a11y` https://github.com/ItsJonQ/g2/pull/125
- [x] `gestures` https://github.com/ItsJonQ/g2/pull/127
- [x] `animations` https://github.com/ItsJonQ/g2/pull/126

**Comment by ItsJonQ** [comment]
Done!
